android/src/main/kotlin/com/chyiiiiiiiiiiiiii/zendesk_messaging/ZendeskMessagingPlugin.kt

```diff
@@ -67,6 +70,13 @@ class ZendeskMessagingPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
                 }
                 zendeskMessaging.logoutUser()
             }
+            "countMessages" -> {
+                if (!isInitialize) {
+                    println("$tag - Messaging needs to be initialized first")
+                    return
+                }
+                result.success(zendeskMessaging.countMessages())
+            }
```

Comment:
- rename to getUnreadMessageCount(), thanks.
- rename isInitialize to isInitialized

Reply:
Fixed in 20b802ed8f51d865d4df359d352885757e5c5a9c